FeedStoreChallenge/Model/ManagedCache.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import CoreData
+
+class ManagedCache: NSManagedObject {
```

Comment:
`ManagedCache` is not supposed to be subclassed, so it should be marked as `final`:
```suggestion
final class ManagedCache: NSManagedObject {
```